systemtest/src/test/java/io/strimzi/systemtest/security/OpaIntegrationST.java

```diff
@@ -99,30 +101,41 @@ void setup() throws Exception {
             .withDefaultConfiguration()
             .install();
 
-        // Install OPA
+        // Install OPA and wait for it to get ready
         KubeResourceManager.get().kubeCmdClient().inNamespace(Environment.TEST_SUITE_NAMESPACE).apply(FileUtils.updateNamespaceOfYamlFile(Environment.TEST_SUITE_NAMESPACE, TestUtils.USER_PATH + "/../systemtest/src/test/resources/opa/opa.yaml"));
+        try {
+            LOGGER.info("Waiting for Open Policy Agent deployment to get ready");
+            KubeResourceManager.get().kubeClient().getClient().apps().deployments().inNamespace(Environment.TEST_SUITE_NAMESPACE).withName("opa").waitUntilReady(120_000L, TimeUnit.MILLISECONDS);
+        } catch (Exception e) {
+            Assertions.fail("OPA deployment is not ready in 120 seconds", e);
+        }
```

Comment:
```suggestion
        DeploymentUtils.waitForDeploymentAndPodsReady(Environment.TEST_SUITE_NAMESPACE, "opa", 1);
```

sorry, maybe instead of that block you can use this one